client/me/billing-history/main.jsx

```diff
@@ -2,7 +2,6 @@
  * External dependencies
  */
 var React = require( 'react' ),
```

Comment:
I was going to do this in a separate PR to make ti easier to review and spot issues in this one.